Reenable tests affected by multiple DestinationResolver beans

When Spring Cloud Stream is compatible with Spring Integration 5.2:
* `PubSubExtendedBindingsPropertiesTests`
* `SampleAppIntegrationTest` in `spring-cloud-gcp-samples/spring-cloud-gcp-pubsub-binder-sample`
*  `SampleAppIntegrationTest` in `spring-cloud-gcp-samples/spring-cloud-gcp-pubsub-polling-binder-sample`
* `LocalSampleAppIntegrationTest` in `spring-cloud-gcp-samples/spring-cloud-gcp-pubsub-bus-config-sample`

There is now a new failure: ```Caused by: java.lang.NoClassDefFoundError: org/springframework/integration/endpoint/Pausable```.

I'll look into that.

--- a/spring-cloud-gcp-pubsub-stream-binder/src/test/java/org/springframework/cloud/gcp/stream/binder/pubsub/properties/PubSubExtendedBindingsPropertiesTests.java
+++ b/spring-cloud-gcp-pubsub-stream-binder/src/test/java/org/springframework/cloud/gcp/stream/binder/pubsub/properties/PubSubExtendedBindingsPropertiesTests.java
@@ -20,7 +20,6 @@ import com.google.cloud.pubsub.v1.Subscriber;
 import com.google.cloud.pubsub.v1.stub.SubscriberStub;
 import com.google.pubsub.v1.Subscription;
 import com.google.pubsub.v1.Topic;
-import org.junit.Ignore;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.mockito.Mockito;
@@ -68,8 +67,7 @@ import static org.mockito.Mockito.when;
 				"spring.cloud.stream.gcp.pubsub.bindings.input.consumer.auto-create-resources=true",
 				"spring.cloud.stream.gcp.pubsub.default.consumer.auto-create-resources=false"
 		})
-@Ignore("Until Spring Cloud Stream is compatible with Spring Integration 5.2 (GH-1648)")
-public class  PubSubExtendedBindingsPropertiesTests {
+public class PubSubExtendedBindingsPropertiesTests {
 
 	@Autowired
 	private ConfigurableApplicationContext context;

--- a/spring-cloud-gcp-samples/spring-cloud-gcp-pubsub-binder-sample/src/test/java/com/example/SampleAppIntegrationTest.java
+++ b/spring-cloud-gcp-samples/spring-cloud-gcp-pubsub-binder-sample/src/test/java/com/example/SampleAppIntegrationTest.java
@@ -23,7 +23,6 @@ import java.util.UUID;
 import org.apache.commons.io.output.TeeOutputStream;
 import org.junit.AfterClass;
 import org.junit.BeforeClass;
-import org.junit.Ignore;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 
@@ -51,7 +50,6 @@ import static org.junit.Assume.assumeThat;
 		"spring.cloud.stream.bindings.input.destination=sub1",
 		"spring.cloud.stream.bindings.output.destination=sub1" })
 @DirtiesContext
-@Ignore("Until Spring Cloud Stream is compatible with Spring Integration 5.2 (GH-1648)")
 public class SampleAppIntegrationTest {
 
 	@Autowired

--- a/spring-cloud-gcp-samples/spring-cloud-gcp-pubsub-bus-config-sample/spring-cloud-gcp-pubsub-bus-config-sample-test/src/test/java/com/example/LocalSampleAppIntegrationTest.java
+++ b/spring-cloud-gcp-samples/spring-cloud-gcp-pubsub-bus-config-sample/spring-cloud-gcp-pubsub-bus-config-sample-test/src/test/java/com/example/LocalSampleAppIntegrationTest.java
@@ -30,7 +30,6 @@ import org.awaitility.Awaitility;
 import org.junit.After;
 import org.junit.AfterClass;
 import org.junit.BeforeClass;
-import org.junit.Ignore;
 import org.junit.Test;
 
 import org.springframework.web.client.RestTemplate;
@@ -50,7 +49,6 @@ import static org.junit.Assume.assumeThat;
  *
  * @since 1.2
  */
-@Ignore("Until Spring Cloud Stream is compatible with Spring Integration 5.2 (GH-1648)")
 public class LocalSampleAppIntegrationTest {
 
 	static final String CONFIG_DIR = "/tmp/config";

--- a/spring-cloud-gcp-samples/spring-cloud-gcp-pubsub-polling-binder-sample/src/test/java/com/example/SampleAppIntegrationTest.java
+++ b/spring-cloud-gcp-samples/spring-cloud-gcp-pubsub-polling-binder-sample/src/test/java/com/example/SampleAppIntegrationTest.java
@@ -26,7 +26,6 @@ import org.apache.commons.io.output.TeeOutputStream;
 import org.awaitility.Awaitility;
 import org.junit.AfterClass;
 import org.junit.BeforeClass;
-import org.junit.Ignore;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 
@@ -54,7 +53,6 @@ import static org.junit.Assume.assumeThat;
 		"spring.cloud.stream.bindings.input.destination=sub1",
 		"spring.cloud.stream.bindings.output.destination=sub1" })
 @DirtiesContext
-@Ignore("Until Spring Cloud Stream is compatible with Spring Integration 5.2 (GH-1648)")
 public class SampleAppIntegrationTest {
 
 	@Autowired

